src/server/auth/server/auth.go

```diff
@@ -19,19 +20,28 @@ import (
 )
 
 const (
-	tokensPrefix = "/pach-tokens"
-	aclsPrefix   = "/acls"
+	tokensPrefix = "/auth/tokens"
+	aclsPrefix   = "/auth/acls"
+	adminsPrefix = "/auth/admins"
 
 	defaultTokenTTLSecs = 24 * 60 * 60
 	authnToken          = "authn-token"
 )
 
 type apiServer struct {
 	protorpclog.Logger
-	etcdClient  *etcd.Client
-	tokenPrefix string
+	etcdClient *etcd.Client
+
+	// This atomic variable stores a boolean flag that indicates
+	// whether the auth service has been activated.
+	activated atomic.Value
```

Comment:
Good point.  We might still keep a local variable as a cache, since we don't really support "deactivation".  But yeah I agree that we should store the activation state in etcd somehow.  Will make the change.